version_test.go

```diff
@@ -1,14 +1,14 @@
-// +build unit
-
 package main
 
 import (
 	"testing"
 
 	"github.com/almighty/almighty-core/app/test"
+	skipper "github.com/almighty/almighty-core/test"
 )
 
 func TestAuthorizeLoginOK(t *testing.T) {
+	skipper.SkiptTestIfNotUnitTest(t)
```

Comment:
Maybe just my personal preference, but.. Wouldn't it read better if it described what it is intended to be, as oppose to what it will do if a certain something.

e.g. test.Integration() test.Unit()

Or maybe describes something about the 'resources' it needs. test.Require(Database, ExternalResource, Mail) etc etc ?